src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3540/EOF1_CallsFiller.yml

```diff
@@ -126,54 +116,44 @@ EOF1_Calls:
       balance: 0
       # EOF code CALLs legacy code
       # code: ':yul { sstore(0, call(100000, 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b, 0, 0, 0, 0, 0)) sstore(1, 1) stop() }'
-      code: ':raw 0xEF000101002D006000600060006000600073c94f5374fce5edbc8e2a8697c15331677e6ebf0b620186a0f1600055600160015500'
-      nonce: 0
-      storage: {}
-    314f5374fce5edbc8e2a8697c15331677e6ebf0b:
-      balance: 0
-      # EOF code CALLCODEs legacy code
-      # code: ':yul { sstore(0, callcode(100000, 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b, 0, 0, 0, 0, 0)) sstore(1, 1) stop() }'
-      code: ':raw 0xEF000101002D006000600060006000600073c94f5374fce5edbc8e2a8697c15331677e6ebf0b620186a0f2600055600160015500'
+      code: ':raw 0xef0001010004020001002d03000000000000076000600060006000600073c94f5374fce5edbc8e2a8697c15331677e6ebf0b620186a0f1600055600160015500'
       nonce: 0
       storage: {}
     324f5374fce5edbc8e2a8697c15331677e6ebf0b:
       balance: 0
       # EOF code DELEGATECALLs legacy code
       # code: ':yul { sstore(0, delegatecall(100000, 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b, 0, 0, 0, 0)) sstore(1, 1) stop() }'
-      code: ':raw 0xEF000101002B00600060006000600073c94f5374fce5edbc8e2a8697c15331677e6ebf0b620186a0f4600055600160015500'
+      code: ':raw 0xef0001010004020001002b0300000000000006600060006000600073c94f5374fce5edbc8e2a8697c15331677e6ebf0b620186a0f4600055600160015500'
       nonce: 0
       storage: {}
     334f5374fce5edbc8e2a8697c15331677e6ebf0b:
       balance: 0
       # EOF code STATICCALLs non-pure legacy code
       # code: ':yul { sstore(0, staticcall(100000, 0xc94f5374fce5edbc8e2a8697c15331677e6ebf0b, 0, 0, 0, 0)) sstore(1, 1) stop() }'
-      code: ':raw 0xEF000101002B00600060006000600073c94f5374fce5edbc8e2a8697c15331677e6ebf0b620186a0fa600055600160015500'
+      code: ':raw 0xef0001010004020001002b0300000000000006600060006000600073c94f5374fce5edbc8e2a8697c15331677e6ebf0b620186a0fa600055600160015500'
       nonce: 0
       storage: {}
     344f5374fce5edbc8e2a8697c15331677e6ebf0b:
       balance: 0
       # EOF code STATICCALLs legacy code
       # code: ':yul { sstore(0, staticcall(100000, 0xd94f5374fce5edbc8e2a8697c15331677e6ebf0b, 0, 0, 0, 1)) sstore(1, mload(0)) sstore(2, 2) stop()}'
-      code: ':raw 0xEF000101003100600160006000600073d94f5374fce5edbc8e2a8697c15331677e6ebf0b620186a0fa600055600051600155600260025500'
+      code: ':raw 0xef000101000402000100310300000000000006600160006000600073d94f5374fce5edbc8e2a8697c15331677e6ebf0b620186a0fa600055600051600155600260025500'
```

Comment:
Ori is working on a script that will convert normal yul into eof code. But it is good to have it defined explicitly here for now

Reply:
Solidity already compiles yul to eof in a branch, see https://github.com/ethereum/tests/pull/1103#issuecomment-1366789807

Reply:
> Ori is working on a script that will convert normal yul into eof code. But it is good to have it defined explicitly here for now

I gave up on that idea (it was to convert bytecode, not yul). It's a lot harder than it appears, because of the way jumps changed and the way you need to track maxStackHeight.